framework/src/application/logger/logger.ts

```diff
@@ -86,7 +86,31 @@ class ConsoleLog {
 				);
 			}
 			if (Object.keys(others).length > 0) {
-				log += util.format('%s \n', JSON.stringify(others, undefined, ' '));
+				log += util.format(
+					'%s \n',
+					JSON.stringify(
+						others,
+						(_, val) => {
+							if (typeof val === 'object') {
+								for (const k of Object.keys(val)) {
+									// eslint-disable-next-line
+									if (typeof val[k] === 'bigint') {
+										// eslint-disable-next-line
+										val[k] = val[k].toString();
+									}
+									// eslint-disable-next-line
+									if (Buffer.isBuffer(val[k])) {
+										// eslint-disable-next-line
+										val[k] = val[k].toString('base64');
```

Comment:
I am not a big fan of that approach but until NodeJS natively support we can do something like this: 

```
Buffer.prototype.toJSON = function () { return this.toString('base64'); }
BigInt.prototype.toJSON = function () { return this.toString(); }
```

This way it will will be done automatically when ever we use `JSON.stringify`. 

https://github.com/cyberphone/es6-bigint-json-support#21-rfc-mode-serialization

Reply:
I would prefer not to overwrite the prototype, and handle them separately.
Especially Buffer does have toJSON implementation